Listener/KernelListener.php

```diff
@@ -58,14 +62,17 @@ class KernelListener
 
     /**
      * @param string $backendUrl
+     * @param null|bool $forceSecureRedirect
      * @param array $crawlerUserAgents
      * @param array $ignoredExtensions
      * @param array $whitelistedUrls
      * @param array $blacklistedUrls
      * @param ClientInterface $httpClient
+     * @param EventDispatcherInterface $eventDispatcher
      */
     public function __construct(
         $backendUrl,
+        $forceSecureRedirect,
```

Comment:
Why putting the new argument, in middle of the argument list ? It's a BC break. The tests are not passing since this introduction